priv/repo/migrations/20200805190647_create_hub_invites_table.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Ret.Repo.Migrations.CreateHubInvitesTable do
+  use Ecto.Migration
+
+  def change do
+    Ret.HubInvite.State.create_type()
+
+    create table(:hub_invites, primary_key: false) do
+      add(:hub_invite_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true)
+      add(:hub_invite_sid, :string, null: false)
+      add(:hub_id, :bigint, null: false)
+      add(:state, :hub_invite_state, null: false, default: "active")
+
+      timestamps()
+    end
+
+    create(index(:hub_invites, [:hub_invite_sid], unique: true))
```

Comment:
Should we also index by `hub_id`?

Reply:
I haven't explicitly used a lookup by `hub_id`, so I generally avoid adding indices unless they're evidently needed.